src/main/java/org/cyclonedx/model/ExternalReference.java

```diff
@@ -66,6 +66,8 @@
         BUILD_META("build-meta"),
         @JsonProperty("build-system")
         BUILD_SYSTEM("build-system"),
+        @JsonProperty("release_notes")
+        RELEASE_NOTES("release_notes"),
```

Comment:
```suggestion
        @JsonProperty("release-notes")
        RELEASE_NOTES("release-notes"),
```
https://github.com/CycloneDX/specification/blob/1.4/schema/bom-1.4.xsd#L776
https://github.com/CycloneDX/specification/blob/1.4/schema/bom-1.4.schema.json#L882